IOSAccessAssessment/Views/AnnotationView.swift

```diff
@@ -52,29 +61,35 @@ struct AnnotationView: View {
                     
                     HStack {
                         Spacer()
-                        VStack {
-                            ForEach(0..<options.count, id: \.self) { optionIndex in
+                        VStack(spacing: 10) {
+                            ForEach(options, id: \.self) { option in
                                 Button(action: {
-                                    // Toggle selection
-                                    if selectedOptionIndex == optionIndex {
-                                        selectedOptionIndex = nil
-                                    } else {
-                                        selectedOptionIndex = optionIndex
+                                    selectedOption = (selectedOption == option) ? nil : option
```

Comment:
Having an inline function is fine for now. 
But when we are working on the API Transmission, we should ideally have a separate function for this Button's on click action